### ISSUE TITLE
Adding WFS data from Micka doesnt work

### ISSUE THREAD

**Issue by FilipLeitner** [hslayers/hslayers-ng#1128]
Adding WFS layer through Micka datasource endpoint fails on missing name/title.
![image](https://user-images.githubusercontent.com/44566616/95491665-31336f00-099a-11eb-8dd6-dff6646612ca.png)

When wfs data is added to map  synchronize property is automaticaly set to true here:
https://github.com/hslayers/hslayers-ng/blob/develop/components/add-layers/vector/VectorLayerDescriptor.js#L71
..so we automatically try to sync with Layman

Applies for both version 1.24 and 2.

**Comment by fzadrazil** [comment]
I think the synchronization should be turned off in this case.

**Comment by raitisbe** [comment]
We will probably have to move the wfs feature pushing and pulling to some separate module, which deals with wfs layers in general, not only the ones served through layman and leave in layman.service just the checking of weather the layer exists and creation of the layer.

**Comment by raitisbe** [comment]
Regarding the 'synchronize' property we need to have something like this to tell if the layer needs to be synced and is not just a plain ordinary vector layer.  The endpoint doesn't necessary need to be layman, but we should then also keep track of the endpoint url in some other layer (Source better maybe) property.

**Comment by FilipLeitner** [comment]
Made temporary workaround 5bd06d88e9ec7adf5461b68d8b3da976980c5509. Not sure it works tho

**Comment by FilipLeitner** [comment]
https://github.com/hslayers/hslayers-ng/pull/1235
https://github.com/hslayers/hslayers-ng/pull/1224
